### PR TITLE
NEPT-1071: Implement the new header search bars component

### PR DESCRIPTION
**Pull request by gillesdeudon**
### Description

Implement the new header search bars component

### Change log

- Added: Add dependency of nexteuropa_europa_search feature to cce_basic_config
- Added: Force the activation of nexteuropa_europa_search feature
- Changed: Add a title to the QueryText textfield of the "nexteuropa_europa_search" form

### Commands

```sh
drush updatedb
```

### CODE REVIEW

**Comment by imanoleguskiza** [comment]
👍

File: tests/features/search/europa_search.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@api @javascript
```
Comment (voidtek) [inline tests/features/search/europa_search.feature]: Why we need the tag @javascript?

File: tests/features/search/europa_search.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    the results is displayed on the "Europa Search" interface
+    Given I am on the homepage
+    And I fill in "Search this website" with "investments"
+    And I press "Search"
+    # Control that we are on the Europa Search interface and that the search work.
+    # We control the minimum to ensure the feature runs correctly.
+    Then the url should match "/geninfo/query/resultaction.jsp"
+    And the "queryText" field should contain "investments"
+    And I should see "Search options"
+    And I should not see "No results found"
```
Comment (voidtek) [inline tests/features/search/europa_search.feature]: I think that this two last steps are not useful for this scenario (out of scope)

Comment (gillesdeudon) [inline tests/features/search/europa_search.feature]: Not for me. it ensures that the europa search reacts correctly to query.

File: tests/features/search/europa_search.feature (hunk 1 above)
Comment (gillesdeudon) [inline tests/features/search/europa_search.feature]: Selenium?

Comment (voidtek) [inline tests/features/search/europa_search.feature]: I do not see nothing in the test that needs this tag.

File: tests/features/search/europa_search.feature (hunk 2 above)
Comment (voidtek) [inline tests/features/search/europa_search.feature]: All tests must be independent from the outside

**Comment by Fefaine** [comment]
👍

**Pull request merged**